bucket/gnupg.json

```diff
@@ -1,15 +1,30 @@
 {
     "homepage": "https://www.gnupg.org/",
+    "description": "An open source implementation of PGP standard, can encrypt and sign data and communications.",
```

Comment:
This will conflict with https://github.com/ScoopInstaller/Main/blob/7c1a4b5a681d16b950c4a0776c48e3a94d6f5bfc/bucket/gnupg.json#L4